packages/protobuf-test/extra/msg-json-names.proto

```diff
@@ -22,9 +22,9 @@ message JsonNamesMessage {
     repeated string repeated_scalar_field = 2 [json_name = "repeatedScalarFieldJsonName"];
 
     string a = 3 [json_name = "sameJsonName"];
-    string b = 4 [json_name = "sameJsonName"];
+    string b = 4 [json_name = "b"];
```

Comment:
The fields 3 to 6 were there to show name clashes. If you simply change them to avoid conflicts, there is no longer any point in having them here. This means the code is really difficult to understand because it has no point. 

Please make sure that the name clashes shown by these fields are addressed by a compiler error or warning now, and remove them.